yarn.lock

```diff
@@ -4775,6 +4775,15 @@ __metadata:
   languageName: unknown
   linkType: soft
 
+"@calcom/synthflow@workspace:packages/app-store/synthflow":
```

Comment:
Can you revert yarn.lock changes, since you haven't added/modified any package these are unnecessary?

https://cal.com/docs/knowledgebase/contributors-guide#guidelines-for-committing-yarn-lockfile

Reply:
Sure sir , I haven't added any code to packages.json but on performing yarn dev it is updating the yarn file as you can see here.
I tried removing it but it is giving this error
![Screenshot 2024-08-31 162415](https://github.com/user-attachments/assets/8864d7f8-f36c-4f20-997f-9d52e15845cd)

Reply:
It may be the changes made by admin 
https://github.com/calcom/cal.com/commit/b576fc8c268a0cc82d56e9a514def5ea7e2f0b29